x/merkledb/db.go

```diff
@@ -422,6 +422,37 @@ func (db *merkleDB) Close() error {
 	return db.baseDB.Put(cleanShutdownKey, hadCleanShutdown)
 }
 
+func (db *merkleDB) PrefetchKeys(keys [][]byte) error {
```

Comment:
I'm planning to call this per key as they are updated...would it make sense to have this only accept a key at each time?

Reply:
I could alternatively replace my prefetch loop with this but I think only ensuring we have the path for modified values will be more efficient.

Reply:
I can create both.